xbmc/LangInfo.h

```diff
@@ -76,9 +99,28 @@ class CLangInfo : public ISettingCallback
   const std::string& GetTimeZone() const;
 
   const std::string& GetRegionLocale() const;
-  const std::string GetLanguageLocale(bool twochar = false) const;
 
-  bool ForceUnicodeFont() const { return m_currentRegion->m_forceUnicodeFont; }
+  /*!
+  \brief Returns the full locale of the current language.
+  */
+  const std::string& GetLocale() const;
+
+  /*!
+   \brief Returns the two character ISO 639-1 language code of the current language.
+   */
+  const std::string& GetLanguageLocale() const;
+
+  /*!
+   \brief Returns the <ll>_<TT> locale of the current language.
+   */
+  const std::string& GetLanguageTerritoryLocale() const;
```

Comment:
I'm still in the process of cleaning this stuff up a bit. The problem is that most sane names are already taken by stuff that we can't remove yet because we are lacking proper/better region handling so it can't be removed right now and is out of the scope of this PR.

And `GetLocale()` and `GetLanguageTerritoryLocale()` won't necessarily return the same thing as the full locale would be `en_GB.UTF-8` (just as an example, we don't have this particular locale in use) but we are often only be interested in `en_GB`.